test/tsconfig.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Just a question - is it different configs per production and tests?

Reply:
Good point - they are the same, and `tsconfig.json` supports inheriting config, so DRYied them up to common `tsconfig.json`.